tldw_Server_API/tests/Admin/test_admin_tools_endpoint.py

```
@@ -0,0 +1,48 @@
+                    {"labels": {"module": "alpha", "tool": "search"}, "count": 2, "avg": 0.1},
+                    {"labels": {"module": "alpha", "tool": "search"}, "count": 1, "avg": 0.2},
+                    {"labels": {"module": "alpha", "tool": "summarize"}, "count": 1, "avg": 0.4},
+                    {"labels": {"module": "beta", "tool": "lookup"}, "count": 3, "avg": 0.05},
+                ]
+            }
+        }
+
+
+@pytest.mark.asyncio
+async def test_get_mcp_tool_usage_aggregates_metrics_from_label_groups(monkeypatch: pytest.MonkeyPatch) -> None:
+    fake_module = types.SimpleNamespace(get_metrics_collector=lambda: _FakeCollector())
```

Ruff (0.15.7)

[warning] Lambda may be unnecessary; consider inlining inner function

Inline function call

(PLW0108)